praw/models/reddit/modmail.py

```diff
@@ -182,17 +182,33 @@ def highlight(self):
         """
         self._reddit.post(API_PATH["modmail_highlight"].format(id=self.id))
 
-    def mute(self):
+    def mute(self, duration=None):
         """Mute the non-mod user associated with the conversation.
 
+        :param duration: Duration of mute in days. Valid options are 3, 7, or 28.
+            Defaults to 3 days. (default: None)
```

Comment:
I'm confused about the default here. Does `duration=None` mean a 3 day mute? If so, I think it makes sense to instead have the default be `duration=3`.

Reply:
I choose this route because I felt it was simpler. The reasons why I did it this way was because it require recording the other mute cassette again and reddit doesn't require the the duration to be explicitly set. I guess I could check if it is 3 and only pass params it is not 3.

Reply:
That sounds like a fine compromise. If Reddit ever changes their default, then we'll have to update it, but that is probably unlikely in the near future. 😄